apps/condo/domains/billing/schema/DiscoverServiceConsumersService.js

```diff
@@ -0,0 +1,75 @@
+/**
+ * Generated by `createservice billing.DiscoverServiceConsumersService --type mutations`
+ */
+
+const { get, _ } = require('lodash')
+
+const { getLogger } = require('@open-condo/keystone/logging')
+const { GQLCustomSchema } = require('@open-condo/keystone/schema')
+
+const access = require('@condo/domains/billing/access/DiscoverServiceConsumersService')
+const { BillingAccount } = require('@condo/domains/billing/utils/serverSchema')
+const { Resident, ServiceConsumer } = require('@condo/domains/resident/utils/serverSchema')
+const logger = getLogger('DiscoverServiceConsumersService')
+
+const DiscoverServiceConsumersService = new GQLCustomSchema('DiscoverServiceConsumersService', {
+    types: [
+        {
+            access: true,
+            type: 'input DiscoverConsumersServiceInput { dv: Int!, sender: SenderFieldInput!, address: String!, unitName: String!, unitType: String!, billingAccountId: ID, residentId: ID }',
+        },
+        {
+            access: true,
+            type: 'type DiscoverConsumersServiceOutput { status: String!, createdServiceConsumersTotal: Int! }',
+        },
+    ],
+    
+    mutations: [
+        {
+            access: access.canDiscoverConsumersService,
+            schema: 'discoverConsumersService(data: DiscoverConsumersServiceInput!): DiscoverConsumersServiceOutput',
```

Comment:
It appears to be language-friendly, but we have a model named ServiceConsumer. And typically use a nameing pattern for mutations like `<verb><model-name>`, such as createUser, `updateServiceConsumer`, and `registerServiceConsumer`. It is more appropriate to follow the same pattern here.